opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -897,15 +899,15 @@ namespace Opm {
 
 
     template<typename TypeTag>
-    void
+    bool
     BlackoilWellModel<TypeTag>::
     assembleImpl(const int iterationIdx,
                  const double dt,
                  const std::size_t recursion_level,
                  DeferredLogger& local_deferredLogger)
     {
 
-        const auto [network_changed, network_imbalance] = updateWellControls(local_deferredLogger, /* check group controls */ true);
+        auto [group_well_changed, network_changed, network_imbalance] = updateWellControls(local_deferredLogger);
```

Comment:
The variable `group_well_changed` here is essentially the same as `well_group_target_violated` in the calling function (and the SimulatorReport), why use different names? It is also not 100% clear that the names mean the same thing.

Reply:
rename to well_group_control_changed